apps/services/auth-public-api/src/app/modules/delegations/meDelegations.controller.ts

```diff
@@ -167,7 +174,7 @@ export class MeDelegationsController {
         namespace,
         action: 'update',
         resources: (delegation) => delegation?.id ?? '',
-        meta: { fields: Object.keys(delegation) },
+        meta: { scopes: delegation.scopes },
```

Comment:
Can the update audit include which scopes were deleted in addition to which were saved?

Reply:
Excellent idea. I will add it as two sperate keys: `added` and `deleted`.

Reply:
After rethinking this, it would introduce a new DB query to get the delegation as is before the update. The input or the result of the update only contains how the scopes should be now.

I'm hesitant to do this, if we simply audit the values from the DTO, you would be able to query the audit log of the last two updates and reviews those list to see new and deleted scopes.